test/test_helper.rb

```diff
@@ -3,6 +3,11 @@
   add_filter "lib/tasks"
   add_filter "lib/lograge"
 
+  # to be deleted after initial deploy of mailer migration to AM/AJ
```

Comment:
@martinemde ^

Reply:
oh, thank you. I didn't catch this part.